gnbug: Add gnbug.

arunisaac opened the pull request:
Hi all,

I have created a simple command-line program to improve life in our
issue tracker. Currently the program can
- list issues
- list issues filtered by assignee
- list all issues created or updated since a certain date

I hope this is useful. Feedback welcome!

Regards,
Arun

* gnbug: New file.
* README.gmi (gnbug): New section.

arunisaac committed:
gnbug: Add gnbug.

* gnbug: New file.
* README.gmi (gnbug): New section.

robwwilliams commented:
Thanks Arun.

On Sat, Sep 18, 2021 at 12:41 PM Arun Isaac ***@***.***>
wrote:

> Hi all,
>
> I have created a simple command-line program to improve life in our
> issue tracker. Currently the program can
>
>    - list issues
>    - list issues filtered by assignee
>    - list all issues created or updated since a certain date
>
> I hope this is useful. Feedback welcome!
>
> Regards,
> Arun
>
>    - gnbug: New file.
>    - README.gmi (gnbug): New section.
>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/genenetwork/gn-gemtext-threads/pull/1
> Commit Summary
>
>    - gnbug: Add gnbug.
>
> File Changes
>
>    - *M* README.gmi
>    <https://github.com/genenetwork/gn-gemtext-threads/pull/1/files#diff-40125e512e6859e64a4b861378800eed5ca891fd8de215c4a0ab08886bee19c7>
>    (21)
>    - *A* gnbug
>    <https://github.com/genenetwork/gn-gemtext-threads/pull/1/files#diff-3c7a6f5ee07ecc138a930e688946e9904a04afb7254d9e08f57544850835b297>
>    (178)
>
> Patch Links:
>
>    - https://github.com/genenetwork/gn-gemtext-threads/pull/1.patch
>    - https://github.com/genenetwork/gn-gemtext-threads/pull/1.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/genenetwork/gn-gemtext-threads/pull/1>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC55V7HUUVAXBLNJCXM3CXLUCTFN7ANCNFSM5EJNEZOQ>
> .
>
-- 
Rob

Robert W. Williams, Ph.D.
Chair: Department of Genetics, Genomics and Informatics
University of Tennessee Health Science Center
71 S Manassas St, Memphis TN 38103-3310,  TSRB Room 405
Tele/Cell: 901 604 4752
EMAIL: ***@***.***  or  ***@***.***

arunisaac committed:
gnbug: Add <issue> type.

* gnbug: Import (srfi srfi-9).
(<issue>): New type.
(issues): New function.
(main): Use <issue> type.

arunisaac committed:
gnbug: Extract title.

* gnbug (<issue>): Add title field.
(issues): Initialize title and assigned fields from hashtable.
(assignees): Delete function.
(file-details): New function.
(main): Print title in issue listing.

arunisaac committed:
gnbug: Extract creator and created date.

* gnbug (<issue>): Add creator, created-date and created-relative-date
fields.
(issues): Initialize creator, created-date and created-relative-date
fields.
(creation-details): New function.
(main): Print creator and created-relative-date in issue listing.

arunisaac committed:
gnbug: Sort issues by creation date, newest first.

* gnbug (issues): Sort issues by creation date, newest first.

arunisaac commented:
I have improved the issue listing, printing creator, creation date, title, and sorting issues newest first.

BonfaceKilz reviewed (approved):
LGTM! @arunisaac thanks for this!
